lib/dl_zitadel/dl_zitadel/middlewares/models.py

```diff
@@ -29,3 +33,11 @@ def get_cookies(
         target: typing.Optional[dl_api_commons_base_models.AuthTarget] = None,
     ) -> dict[dl_constants_api_constants.DLCookies, str]:
         return {}
+
+
+@attr.s(frozen=True)
+class AuthResult:
+    data: ZitadelAuthData = attr.ib()
+    user_id: typing.Optional[str] = attr.ib(default=None)
+    user_name: typing.Optional[str] = attr.ib(default=None)
```

Comment:
non optional

Reply:
> non optional

it's optional in IntrospectResult so i have to do it optional here

Reply:
hmmm.... i assume they're not if active=True, but ok, agreed